src/main/java/me/coley/recaf/workspace/JavaResource.java

```diff
@@ -352,26 +354,26 @@ public void invalidate() {
 					code.analyze();
 					map.put(code.getInternalName(), code);
 				} catch(SourceCodeException ex) {
-					error(ex, "Failed to parse source: {} in {}", name, file);
+					error(ex, "Failed to parse source: {} in {}", name, path);
 				}
 			}
 		}
 		return map;
 	}
 
 	/**
-	 * @param file
-	 * 		File containing documentation.
+	 * @param path
+	 * 		Path containing documentation.
 	 *
 	 * @return Map of class names to their documentation.
 	 *
 	 * @throws IOException
 	 * 		When the file could not be fetched or parsed.
 	 */
-	protected Map<String, Javadocs> loadDocs(File file) throws IOException {
-		Map<String, Javadocs> map = new HashMap<>();
+	protected Map<String, Javadocs> loadDocs(Path path) throws IOException {
+		Map<String, Javadocs> map = new HashMap<>(16, 1F);
```

Comment:
I don't see why specifying the initial capacity or load factor is necessary. 
Most javadoc archives have much more than 16 files too.